charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -1127,9 +1127,9 @@ kubeEtcd:
     # -- Enable service for ETCD metrics scraping
     enabled: true
     # -- ETCD service port
-    port: 2379
+    port: 2381
```

Comment:
Won't this break etcd scraping for older versions where the port is still 2379?

Reply:
updated minikube values port instead, since GKE still relies on 2381